src/main/java/spireMapOverhaul/zones/example/monsters/DreadMoth.java

```diff
@@ -0,0 +1,105 @@
+package spireMapOverhaul.zones.example.monsters;
```

Comment:
I recall us discussing that rather than have separate top level packages for powers/cards/monsters/etc., each zone should just have those classes in their zone package. That's what I've done here.

For all the monsters I've added, since they're adapted from my existing act mods, I did not translate them to use the various methods in `AbstractSMOMonster` that are intended to make setting up moves easier.